src/ComputeSharp/Properties/ILLink.Substitutions.xml

```diff
@@ -0,0 +1,18 @@
+<linker>
+  <assembly fullname="ComputeSharp">
+    <type fullname="Configuration">
+
+      <!-- COMPUTESHARP_ENABLE_DEBUG_OUTPUT switch -->
+      <method signature="System.Boolean get_IsDebugOutputEnabled()" body="stub" value="false" feature="COMPUTESHARP_ENABLE_DEBUG_OUTPUT" featurevalue="false"/>
+      <method signature="System.Boolean get_IsDebugOutputEnabled()" body="stub" value="true" feature="COMPUTESHARP_ENABLE_DEBUG_OUTPUT" featurevalue="true"/>
+
+      <!-- COMPUTESHARP_ENABLE_DEVICE_REMOVED_EXTENDED_DATA switch -->
+      <method signature="System.Boolean get_IsDeviceRemovedExtendedDataEnabled()" body="stub" value="false" feature="COMPUTESHARP_ENABLE_DEVICE_REMOVED_EXTENDED_DATA" featurevalue="false"/>
+      <method signature="System.Boolean get_IsDeviceRemovedExtendedDataEnabled()" body="stub" value="true" feature="COMPUTESHARP_ENABLE_DEVICE_REMOVED_EXTENDED_DATA" featurevalue="true"/>
+
+      <!-- COMPUTESHARP_DISABLE_GPU_TIMEOUT switch -->
```

Comment:
Yeah I was thinking the same as well to avoid the double negation, just figured I'd do that in a separate PR since that's technically unrelated from making the feature switches trimmable. Good point though! 👍